test/testCasePoints.test.ts

```diff
@@ -13,6 +13,16 @@ const azureConfig: IAzureConfig = {
   runName: 'sample',
 }
 
+const azureConfigWithConfigName: IAzureConfig = {
+  pat: 'v3e3pg4njfmzzcuoxwb4vja3trwbyafim4x4oww7sbcfbopfb3bq',
```

Comment:
Can we have the token as part of env variable?

Reply:
yes, moved it to env.